### PR TITLE
update mdn teams in Localize

### PR DESCRIPTION
**Pull request by yin1999**
#### Summary

update mdn teams in MDN/Contribute/Localize

#### Metadata

- [x] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by dipikabh** [comment]
Hello @mdn/localization-team-leads, please take a look at the changes to verify if all updates look good. Thank you!

**Review by hochan222 — APPROVED** [review]
LGTM for ko locale :)

**Comment by schalkneethling** [comment]
Friendly ping for review @dipikabh - Thanks

File: files/en-us/mdn/contribute/localize/index.md [hunk 1]
```diff
@@ -39,17 +39,17 @@ We have frozen all localized content (meaning that we won't accept any edits to
...
-- Current contributors: [armanpwnz](https://github.com/armanpwnz), [captainspring](https://github.com/captainspring), [mpstv](https://github.com/mpstv), [myshov](https://github.com/myshov), [Saionaro](https://github.com/Saionaro), [sashasushko](https://github.com/sashasushko), [lex111](https://github.com/lex111)
+- Current contributors: [armanpwnz](https://github.com/armanpwnz), [captainspring](https://github.com/captainspring), [myshov](https://github.com/myshov), [Saionaro](https://github.com/Saionaro), [sashasushko](https://github.com/sashasushko), [lex111](https://github.com/lex111)
 
 ### Spanish (es)
 
 - Discussions: [Matrix (#mdn-l10n-es channel)](https://chat.mozilla.org/#/room/#mdn-l10n-es:mozilla.org)
-- Current maintainers: [@JuanVqz](https://github.com/JuanVqz), [@dacalderonp](https://github.com/dacalderonp), [@lasr21](https://github.com/lasr21), [@tuxxy](https://github.com/tuxxy)
+- Current maintainers: [@JuanVqz](https://github.com/JuanVqz), [@dacalderonp](https://github.com/dacalderonp), [@davbrito](https://github.com/davbrito), [@Graywolf9](https://github.com/Graywolf9), [@Vallejoanderson](https://github.com/Vallejoanderson)
```
Comment (JuanVqz) [inline files/en-us/mdn/contribute/localize/index.md:52]: :wave: Hey @yin1999, @mdn/yari-content-es has been changing a little bit, this is our team. Thank you! :rocket: 

```suggestion
- Current maintainers: [@JuanVqz](https://github.com/JuanVqz), [@davbrito](https://github.com/davbrito), [@Graywolf9](https://github.com/Graywolf9), [@Vallejoanderson](https://github.com/Vallejoanderson), [@marcelozarate](https://github.com/marcelozarate), [@Jalkhov](https://github.com/Jalkhov)
```

File: files/en-us/mdn/contribute/localize/index.md [hunk 2]
```diff
@@ -39,17 +39,17 @@ We have frozen all localized content (meaning that we won't accept any edits to
...
+- Current contributors: [hochan222](https://github.com/hochan222), [yechoi42](https://github.com/yechoi42), [cos18](https://github.com/cos18), [GwangYeol-Im](https://github.com/GwangYeol-Im), [pje1740](https://github.com/pje1740), [yujo11](https://github.com/yujo11)
 
 ### Russian (ru)
 
 - Discussions: [Matrix (#mdn-l10n-ru channel)](https://chat.mozilla.org/#/room/#mdn-l10n-ru:mozilla.org)
-- Current contributors: [armanpwnz](https://github.com/armanpwnz), [captainspring](https://github.com/captainspring), [mpstv](https://github.com/mpstv), [myshov](https://github.com/myshov), [Saionaro](https://github.com/Saionaro), [sashasushko](https://github.com/sashasushko), [lex111](https://github.com/lex111)
+- Current contributors: [armanpwnz](https://github.com/armanpwnz), [captainspring](https://github.com/captainspring), [myshov](https://github.com/myshov), [Saionaro](https://github.com/Saionaro), [sashasushko](https://github.com/sashasushko), [lex111](https://github.com/lex111)
```
Comment (JuanVqz) [inline files/en-us/mdn/contribute/localize/index.md:47]:
```suggestion
- Current contributors: [@armanpwnz](https://github.com/armanpwnz), [@captainspring](https://github.com/captainspring), [@myshov](https://github.com/myshov), [@Saionaro](https://github.com/Saionaro), [@sashasushko](https://github.com/sashasushko), [@lex111](https://github.com/lex111)
```

File: files/en-us/mdn/contribute/localize/index.md [hunk 3]
```diff
@@ -39,17 +39,17 @@ We have frozen all localized content (meaning that we won't accept any edits to
 ### Korea (ko)
 
 - Discussions: [Kakao Talk (#MDN Korea)](https://open.kakao.com/o/gdfG288c)
-- Current contributors: [hochan222](https://github.com/hochan222), [yechoi42](https://github.com/yechoi42), [cos18](https://github.com/cos18), [GwangYeol-Im](https://github.com/GwangYeol-Im), [pje1740](https://github.com/pje1740), [nKiNk](https://github.com/nKiNk), [yujo11](https://github.com/yujo11)
+- Current contributors: [hochan222](https://github.com/hochan222), [yechoi42](https://github.com/yechoi42), [cos18](https://github.com/cos18), [GwangYeol-Im](https://github.com/GwangYeol-Im), [pje1740](https://github.com/pje1740), [yujo11](https://github.com/yujo11)
```
Comment (JuanVqz) [inline files/en-us/mdn/contribute/localize/index.md:42]:
```suggestion
- Current contributors: [@hochan222](https://github.com/hochan222), [@yechoi42](https://github.com/yechoi42), [@cos18](https://github.com/cos18), [@GwangYeol-Im](https://github.com/GwangYeol-Im), [@pje1740](https://github.com/pje1740), [@yujo11](https://github.com/yujo11)
```

File: files/en-us/mdn/contribute/localize/index.md [hunk 4]
```diff
@@ -19,12 +19,12 @@ We have frozen all localized content (meaning that we won't accept any edits to
...
 
 - Discussions: [Telegram (MDN localization in Brazilian Portuguese)](https://t.me/mdn_l10n_pt_br)
-- Current contributors: [Luisa Migueres](https://github.com/lumigueres), [Julio Sampaio](https://github.com/juliosampaio), [Josiel Rocha](https://github.com/josielrocha), [Clóvis Lima Júnior](https://github.com/clovislima)
+- Current contributors: [Nathália Pissuti](https://github.com/nathipg), [Josiel Rocha](https://github.com/josielrocha), [Clóvis Lima Júnior](https://github.com/clovislima)
 
 ### Chinese (zh-CN, zh-TW)
 
 - Discussions: [Telegram (MozTW L10n channel)](https://moztw.org/community/telegram/)
-- Current contributors: [Irvin](https://github.com/irvin), [t7yang](https://github.com/t7yang), [dibery](https://github.com/dibery)
+- Current contributors: [Irvin](https://github.com/irvin), [t7yang](https://github.com/t7yang), [yin1999](https://github.com/yin1999)
```
Comment (JuanVqz) [inline files/en-us/mdn/contribute/localize/index.md:27]:
```suggestion
- Current contributors: [@Irvin](https://github.com/irvin), [@t7yang](https://github.com/t7yang), [@yin1999](https://github.com/yin1999)
```

File: files/en-us/mdn/contribute/localize/index.md (hunk 1 above)
Comment (yin1999) [reply files/en-us/mdn/contribute/localize/index.md:52]: Thanks for suggestion 😊

File: files/en-us/mdn/contribute/localize/index.md [hunk 5]
```diff
@@ -19,12 +19,12 @@ We have frozen all localized content (meaning that we won't accept any edits to
 ### Brazilian Portuguese (pt-BR)
 
 - Discussions: [Telegram (MDN localization in Brazilian Portuguese)](https://t.me/mdn_l10n_pt_br)
-- Current contributors: [Luisa Migueres](https://github.com/lumigueres), [Julio Sampaio](https://github.com/juliosampaio), [Josiel Rocha](https://github.com/josielrocha), [Clóvis Lima Júnior](https://github.com/clovislima)
+- Current contributors: [Nathália Pissuti](https://github.com/nathipg), [Josiel Rocha](https://github.com/josielrocha), [Clóvis Lima Júnior](https://github.com/clovislima)
```
Comment (JuanVqz) [inline files/en-us/mdn/contribute/localize/index.md:22]: I don't know if add the `at` symbol at the beginning and change their names or keep it as it is.

**Review by JuanVqz — COMMENTED** [review]
I saw that the Spanish team has `at` at the beginning of the name and I was wondering if all teams should have the same  format or we should use the formal name as the pt-BR has, (just to be consistent)

**Comment by yin1999** [comment]
> I saw that the Spanish team has `at` at the beginning of the name and I was wondering if all teams should have the same format or we should use the formal name as the pt-BR has, (just to be consistent)

I'm also not really sure about this. Does anyone have suggestions?

**Comment by SphinxKnight** [comment]
I'd say GitHub does not necessarily present usernames with an @ (see on https://github.com/SphinxKnight) (contrary to Twitter for instance) and that the current page works without an @: https://developer.mozilla.org/en-US/docs/MDN/Contribute/Localize) 

In my (not so strong) opinion, one could go without @ for the sake of GH users' mentions consistency and lightness.

**Review by JuanVqz — APPROVED** [review]

**Review by dipikabh — APPROVED** [review]
Thanks everyone!

**Pull request merged**